docs/src/main/asciidoc/centralized-log-management.adoc

```diff
@@ -30,7 +30,7 @@ mvn io.quarkus:quarkus-maven-plugin:{quarkus-version}:create \
     -DprojectArtifactId=gelf-logging \
     -DclassName="org.acme.quickstart.GelfLoggingResource" \
     -Dpath="/gelf-logging" \
-    -Dextensions="logging-gelf"
```

Comment:
reason for not using noExamples is that the example acutally uses resteasy, right ?

Reply:
That one is using `-Dpath="/gelf-logging"` meaning it was using the resteasy examples in the legacy. So I am just keeping it as it was..